pkg/opssight/ctl_opssight.go

```diff
@@ -120,84 +158,80 @@ func (ctl *HelmValuesFromCobraFlags) AddCobraFlagsToCommand(cmd *cobra.Command,
 	cmd.Flags().SortFlags = false
 
 	// Version
-	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", "2.2.5", "Version of the OpsSight instance\n")
+	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", DefaultFlagTree.Version, "Version of the OpsSight instance\n")
 	if master {
 		cobra.MarkFlagRequired(cmd.Flags(), "version")
 	}
 
 	// Memory
-	cmd.Flags().StringVar(&ctl.flagTree.DeploymentResourcesFilePath, "deployment-resources-file-path", ctl.flagTree.DeploymentResourcesFilePath, "Absolute path to a file containing a list of deployment Resources json structs")
-	cmd.Flags().StringVar(&ctl.flagTree.DefaultCPU, "default-cpu", ctl.flagTree.DefaultCPU, "CPU size of Opssight")
-	cmd.Flags().StringVar(&ctl.flagTree.DefaultMem, "default-memory", ctl.flagTree.DefaultMem, "Memory size of Opssight")
-	cmd.Flags().StringVar(&ctl.flagTree.ScannerCPU, "scanner-cpu", ctl.flagTree.ScannerCPU, "CPU size of Opssight's Scanner")
-	cmd.Flags().StringVar(&ctl.flagTree.ScannerMem, "scanner-memory", ctl.flagTree.ScannerMem, "Memory size of Opssight's Scanner\n")
+	cmd.Flags().StringVar(&ctl.flagTree.DeploymentResourcesFilePath, "deployment-resources-file-path", ctl.flagTree.DeploymentResourcesFilePath, "Absolute path to a file containing a list of deployment Resources json structs\n")
 
 	// Registry Configuration
 	// cmd.Flags().StringVar(&ctl.flagTree.IsUpstream, "is-upstream", ctl.flagTree.IsUpstream, "If true, Upstream images and names will be used [true|false]")
 	cmd.Flags().StringVar(&ctl.flagTree.Registry, "registry", ctl.flagTree.Registry, "Name of the registry to use for images e.g. docker.io/blackducksoftware")
 	cmd.Flags().StringSliceVar(&ctl.flagTree.PullSecrets, "pull-secret-name", ctl.flagTree.PullSecrets, "Only if the registry requires authentication\n")
 	// cmd.Flags().StringSliceVar(&ctl.flagTree.ImageRegistries, "image-registries", ctl.flagTree.ImageRegistries, "List of image registries")
 
-	cmd.Flags().StringVar(&ctl.flagTree.LogLevel, "log-level", ctl.flagTree.LogLevel, "Log level of Opssight\n")
+	cmd.Flags().StringVar(&ctl.flagTree.LogLevel, "log-level", DefaultFlagTree.LogLevel, "Log level of Opssight\n")
 
 	// Black Duck Configuration
 	if master {
 		// During create users can specify files, otherwise they need to use commands like "./synopsysctl update opssight externalhost"
 		cmd.Flags().StringVar(&ctl.flagTree.BlackduckExternalHostsFilePath, "blackduck-external-hosts-file-path", ctl.flagTree.BlackduckExternalHostsFilePath, "Absolute path to a file containing a list of Black Duck External Hosts")
 		cmd.Flags().StringVar(&ctl.flagTree.BlackduckSecuredRegistriesFilePath, "blackduck-secured-registries-file-path", ctl.flagTree.BlackduckSecuredRegistriesFilePath, "Absolute path to a file containing a list of Black Duck Secured Registries")
```

Comment:
Is there a way to deprecate this flag and use the same values to name it as privateDockerSecuredRegistriesFilePath?

Reply:
You just want to change the name? We can do that but I think it should be in a different PR

Reply:
I will handle this in the registries PR that we are going to open